### PR TITLE
create a web font for wo ist markt

### PR DESCRIPTION
**Pull request by tursics**

### CODE REVIEW

**Comment by k-nut** [comment]
why

**Comment by tursics** [comment]
If you want to change the symbol on your map marker (with the used leaflet extension) you must use a symbol font. And here it is: a wo-ist-markt web font.

**Comment by k-nut** [comment]
I guess we could also create regular makers with SVG elements instead of continuing to use the awesome-markers extensions. It sounds to me like we are making life unnecessarily difficult with this.
Or is there a reason that we must stick with the awesome-markers extensions?

Apart from that its cool to see that we have our own font now! Would you care explaining how you made it? :)

**Comment by johnjohndoe** [comment]
@tursics You could add a `README.md` in the `font` folder.

**Comment by tursics** [comment]
We can switch to our own marker code. But life is easier with the awesome-markers extension. It give us markers in different colors (we use this feature), shadows (this too) and possibilities to change the icon on the marker in an easy way. Currently only the basket icon is used. But in the future we want to use different icons for different market types.

When we use icons and colors - it's simpler to use this 3rd party extension

**Comment by tursics** [comment]
@johnjohndoe a readme file was added

**Comment by k-nut** [comment]
@tursics thank you for the explanation. Sounds like sticking with the library is the best option for now 👍

File: assets/font/README.MD [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Wo ist Markt? font
+
+This folder contains the Wo ist Markt? font.
+
+
+## Generation
+
+* prepare your graphic files
```
Comment (johnjohndoe) [inline assets/font/README.MD]: How about using an ordered list? Reads easier, like a receipe.

**Review by johnjohndoe — APPROVED** [review]

**Pull request merged**